src/themes/teams/components/Icon/svg/icons/addPage.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
```

Comment:
we need to mark corresponding icon as exported one on the Processed Icons page in the docs - it is needed to specify value for `exportedAs` field for the corresponding icon from processed set (`canvasaddnewpage`?). 

Please, take a look on this small PR that you could consider as an example for that: https://github.com/stardust-ui/react/pull/585

Reply:
Done